01-chapter1.markdown: grammar and spelling corrections

vlt opened the pull request:
Hi!

While reading progit I made some grammar and spelling corrections.
And to practice I created this first pull request.
Is this the right way to participate?

Daniel

vlt committed:
01-chapter1.markdown: grammar and spelling corrections

jublo commented:
Hello Daniel,

Thank you for your work. We appreciate that you’d like to contribute and make Pro Git better. Yes, sending a pull request is the right way to collaborate!

We’re now proofreading your changes and will update you about the result. Stay tuned!

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -34,14 +34,14 @@ Bild 1-1. Diagramm: Lokale Versionskontrolle
 <!--One of the more popular VCS tools was a system called rcs, which is still distributed with many computers today. Even the popular Mac OS X operating system includes the rcs command when you install the Developer Tools. This tool basically works by keeping patch sets (that is, the differences between files) from one revision to another in a special format on disk; it can then recreate what any ...
 
-Eines der populärsten Versionskontrollsysteme war rcs, und es wird heute immer noch mit vielen Computern ausgeliefert. Z.B. umfasst auch das Betriebssystem Mac OS X den Befehl rcs, wenn Du die Developer Tools installierst. Dieser Befehl arbeitet nach dem Prinzip, dass er für jede Änderung einen Patch (d.h. eine Kodierung der Unterschiede, die eine Änderung an einer oder mehreren Dateien umfasst)  ...
+Eines der populärsten Versionskontrollsysteme war rcs, und es wird heute immer noch mit vielen Computern ausgeliefert. Zum Beispiel umfasst auch das Betriebssystem Mac OS X den Befehl rcs, wenn Du die Developer-Tools installierst. Dieser Befehl arbeitet nach dem Prinzip, dass er für jede Änderung einen Patch (d. h. eine Kodierung der Unterschiede, die eine Änderung an einer oder mehreren Dateien  ...
```
`d. h.`: Is this a non-breaking space?

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -50,18 +50,18 @@ Bild 1-2. Diagramm: Zentralisierte Versionskontrollsysteme
 <!--This setup offers many advantages, especially over local VCSs. For example, everyone knows to a certain degree what everyone else on the project is doing. Administrators have fine-grained control over who can do what; and it’s far easier to administer a CVCS than it is to deal with local databases on every client.-->
 
-Dieser Aufbau hat viele Vorteile gegenüber Lokalen Versionskontrollsystemen. Zum Beispiel weiß jeder mehr oder weniger genau darüber Bescheid, was andere, an einem Projekt Beteiligte gerade tun. Administratoren haben die Möglichkeit, detailliert festzulegen, wer was tun kann. Und es ist sehr viel einfacher, ein CVCS zu administrieren als lokale Datenbanken auf jedem einzelnen Anwenderrechner zu v ...
+Dieser Aufbau hat viele Vorteile gegenüber Lokalen Versionskontrollsystemen. Zum Beispiel weiß jeder mehr oder weniger genau darüber Bescheid, was andere an einem Projekt beteiligte gerade tun. Administratoren haben die Möglichkeit, detailliert festzulegen, wer was tun kann. Und es ist sehr viel einfacher, ein CVCS zu administrieren als lokale Datenbanken auf jedem einzelnen Anwenderrechner zu ve ...
```
Your second change has incorrect grammar in German. `Beteiligte` is an adjective used as noun, so it needs to be capitalised. The removed comma is correct, though.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -50,18 +50,18 @@ Bild 1-2. Diagramm: Zentralisierte Versionskontrollsysteme
 <!--This is where Distributed Version Control Systems (DVCSs) step in. In a DVCS (such as Git, Mercurial, Bazaar or Darcs), clients don’t just check out the latest snapshot of the files: they fully mirror the repository. Thus if any server dies, and these systems were collaborating via it, any of the client repositories can be copied back up to the server to restore it. Every checkout is really a ...
 
-Und an dieser Stelle kommen verteilte Versionskontrollsysteme (DVCS) ins Spiel. In einem DVCS (wie z.B. Git, Mercurial, Bazaar oder Darcs) erhalten Anwender nicht einfach den jeweils letzten Snapshot des Projektes von einem Server: sie erhalten statt dessen eine vollständige Kopie des Repositories. Auf diese Weise kann, wenn ein Server beschädigt wird, jedes beliebige Repository von jedem beliebi ...
+Und an dieser Stelle kommen verteilte Versionskontrollsysteme (DVCS) ins Spiel. In einem DVCS (wie z. B. Git, Mercurial, Bazaar oder Darcs) erhalten Anwender nicht einfach den jeweils letzten Snapshot des Projektes von einem Server: Sie erhalten statt dessen eine vollständige Kopie des Repositorys. Auf diese Weise kann, wenn ein Server beschädigt wird, jedes beliebige Repository von jedem beliebi ...
```
- `d. h.`: Is this a non-breaking space?
- Please amend concatenating `statt dessen` to read `stattdessen`.
- Please amend concatenating `zurück kopiert` to read `zurückkopiert`.
- Please amend concatenating `wieder hergestellt` to read `wiederhergestellt`.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -70,18 +70,18 @@ Bild 1-3. Diagramm: Distribuierte Versionskontrolle
 <!--As with many great things in life, Git began with a bit of creative destruction and fiery controversy. The Linux kernel is an open source software project of fairly large scope. For most of the lifetime of the Linux kernel maintenance (1991–2002), changes to the software were passed around as patches and archived files. In 2002, the Linux kernel project began using a proprietary DVCS system c ...
 
-Wie viele großartige Dinge im Leben entstand Git aus kreativem Chaos und hitziger Diskussion. Der Linux Kernel ist ein Open Source Software Projekt von erheblichem Umfang. Während der gesamten Entwicklungszeit des Linux Kernels von 1991 bis 2002 wurden Änderungen an der Software in Form von Patches (d.h. Änderungen an bestehendem Code) und archivierten Dateien herumgereicht. 2002 began man dann,  ...
+Wie viele großartige Dinge im Leben entstand Git aus kreativem Chaos und hitziger Diskussion. Der Linux-Kernel ist ein Open-Source-Softwareprojekt von erheblichem Umfang. Während eines Großteils der Entwicklungszeit des Linux-Kernels (von 1991 bis 2002) wurden Änderungen an der Software in Form von Patches (d. h. Änderungen an bestehendem Code) und archivierten Dateien herumgereicht. 2002 begann  ...
```
`d. h.`: Is this a non-breaking space?

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -91,20 +91,20 @@ Wie viele großartige Dinge im Leben entstand Git aus kreativem Chaos und hitzig
 * Geschwindigkeit
 * Einfaches Design
-* Gute Unterstützung von nicht-linearer Entwicklung (tausende paralleler Branches, d.h. verschiedener Verzweigungen der Versionen)
+* Gute Unterstützung von nicht-linearer Entwicklung (tausende paralleler Branches, d. h. verschiedener Verzweigungen der Versionen)
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -120,7 +120,7 @@ Bild 1-4. Andere Systeme speichern Daten als Änderungen an einzelnen Dateien ei
 <!--Git doesn’t think of or store its data this way. Instead, Git thinks of its data more like a set of snapshots of a mini filesystem. Every time you commit, or save the state of your project in Git, it basically takes a picture of what all your files look like at that moment and stores a reference to that snapshot. To be efficient, if files have not changed, Git doesn’t store the file again—jus ...
 
-Git sieht Daten nicht in dieser Weise. Stattdessen betrachtet Git seine Daten eher als eine Reihe von Snapshots eines Mini-Dateisystems. Jedes Mal, wenn Du committest (d.h. den gegenwärtigen Status Deines Projektes als eine Version in Git speicherst), sichert Git den Zustand sämtlicher Dateien in diesem Moment („Snapshot“) und speichert eine Referenz auf diesen Snapshot. Um dies möglichst effizie ...
+Git sieht Daten nicht in dieser Weise. Stattdessen betrachtet Git seine Daten eher als eine Reihe von Snapshots eines Mini-Dateisystems. Jedes Mal, wenn Du committest (d. h. den gegenwärtigen Status Deines Projektes als eine Version in Git speicherst), sichert Git den Zustand sämtlicher Dateien in diesem Moment („Snapshot“) und speichert eine Referenz auf diesen Snapshot. Um dies möglichst effizi ...
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -129,14 +129,14 @@ Bild 1-5. Git speichert Daten als eine Historie von Snapshots des Projektes.
 <!--This is an important distinction between Git and nearly all other VCSs. It makes Git reconsider almost every aspect of version control that most other systems copied from the previous generation. This makes Git more like a mini filesystem with some incredibly powerful tools built on top of it, rather than simply a VCS. We’ll explore some of the benefits you gain by thinking of your data this  ...
 
-Dies ist ein wichtiger Unterschied zwischen Git und praktisch allen anderen Versionskontrollsystemen. In Git wurden daher fast alle Aspekte der Versionskontrolle neu überdacht, die in anderen Systemen mehr oder weniger von ihren jeweiligen Vorgängergeneration übernommen worden waren. Git arbeitet im Großen und Ganzen eher wie ein (mit einigen unglaublich mächtigen Werkezeugen ausgerüstetes) Mini- ...
+Dies ist ein wichtiger Unterschied zwischen Git und praktisch allen anderen Versionskontrollsystemen. In Git wurden daher fast alle Aspekte der Versionskontrolle neu überdacht, die in anderen Systemen mehr oder weniger von ihren jeweiligen Vorgängergeneration übernommen worden waren. Git arbeitet im Großen und Ganzen eher wie ein (mit einigen unglaublich mächtigen Werkzeugen ausgerüstetes) Mini-D ...
```
Consider replacing `mächtigen` with `leistungsstarken`.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -144,24 +144,24 @@ Um beispielsweise die Historie des Projektes zu durchsuchen, braucht Git sie nic
 <!--This also means that there is very little you can’t do if you’re offline or off VPN. If you get on an airplane or a train and want to do a little work, you can commit happily until you get to a network connection to upload. If you go home and can’t get your VPN client working properly, you can still work. In many other systems, doing so is either impossible or painful. In Perforce, for exampl ...
 
-Dies bedeutet natürlich außerdem, dass es fast nichts gibt, was Du nicht tun kannst, bloß weil Du gerade offline bist oder keinen Zugriff auf ein VPN hast. Wenn Du im Flugzeug oder Zug ein wenig arbeiten willst, kannst Du problemlos Deine Arbeit committen und Deine Arbeit erst auf den Server pushen (hochladen), wenn Du wieder mit dem Internet verbunden bist. Wenn Du zu Hause bist aber nicht auf d ...
+Dies bedeutet natürlich außerdem, dass es fast nichts gibt, was Du nicht tun kannst, bloß weil Du gerade offline bist oder keinen Zugriff auf ein VPN hast. Wenn Du im Flugzeug oder Zug ein wenig arbeiten willst, kannst Du problemlos Deine Arbeit committen und Deine Arbeit erst auf den Server pushen (hochladen), wenn Du wieder mit dem Internet verbunden bist. Wenn Du zu Hause bist aber nicht auf d ...
```
`z. B.`: Is this a non-breaking space?

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -144,24 +144,24 @@ Um beispielsweise die Historie des Projektes zu durchsuchen, braucht Git sie nic
 <!--Everything in Git is check-summed before it is stored and is then referred to by that checksum. This means it’s impossible to change the contents of any file or directory without Git knowing about it. This functionality is built into Git at the lowest levels and is integral to its philosophy. You can’t lose information in transit or get file corruption without Git being able to detect it.-->
 
-In Git werden Änderungen in Checksummen umgerechnet, bevor sie gespeichert werden. Anschließend werden sie mit dieser Checksumme referenziert. Das macht es unmöglich, dass sich die Inhalte von Dateien oder Verzeichnissen ändern, ohne dass Git das mitbekommt. Git basiert auf dieser Funktionalität und sie ist ein integraler Teil von Gits Philosophie. Man kann Informationen deshalb z.B. nicht währen ...
+In Git werden Änderungen in Checksummen umgerechnet, bevor sie gespeichert werden. Anschließend werden sie mit dieser Checksumme referenziert. Das macht es unmöglich, dass sich die Inhalte von Dateien oder Verzeichnissen ändern, ohne dass Git das mitbekommt. Git basiert auf dieser Funktionalität und sie ist ein integraler Teil von Gits Philosophie. Man kann Informationen deshalb z. B. nicht währe ...
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -212,11 +212,11 @@ Der grundlegend Git Arbeitsprozess sieht in etwa so aus:
 1. Du bearbeitest Dateien in Deinem Arbeitsverzeichnis.
 2. Du markierst Dateien für den nächsten Commit, indem Du Snapshots zur Staging Area hinzufügst.
-3. Du legst den Commit an, wodurch die in der Staging Area vorgemerkten Snapshots dauerhaft im Git Verzeichnis (d.h. der lokalen Datenbank) gespeichert werden.
+3. Du legst den Commit an, wodurch die in der Staging Area vorgemerkten Snapshots dauerhaft im Git-Verzeichnis (d. h. der lokalen Datenbank) gespeichert werden.
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -230,11 +230,11 @@ Lass uns damit anfangen, Git tatsächlich zu verwenden. Der erste Schritt besteh
 <!--If you can, it’s generally useful to install Git from source, because you’ll get the most recent version. Each version of Git tends to include useful UI enhancements, so getting the latest version is often the best route if you feel comfortable compiling software from source. It is also the case that many Linux distributions contain very old packages; so unless you’re on a very up-to-date dis ...
 
-Wenn es Dir möglich ist, empfehlen wir, Git vom Quellcode aus zu installieren, weil Du die jeweils neueste Version erhältst. In der Regel bringt jede Version nützliche Verbesserungen (z.B. am Interface), sodass es sich lohnt die jeweils neueste Version zu verwenden – sofern Du natürlich damit klarkommst, Software aus dem Quellcode zu kompilieren. Viele Linux Distributionen umfassen sehr alte Git  ...
+Wenn es Dir möglich ist, empfehlen wir, Git vom Quellcode aus zu installieren, weil Du die jeweils neueste Version erhältst. In der Regel bringt jede Version nützliche Verbesserungen (z. B. am Interface), sodass es sich lohnt, die jeweils neueste Version zu verwenden – sofern Du natürlich damit klarkommst, Software aus dem Quellcode zu kompilieren. Viele Linux-Distributionen umfassen sehr alte Gi ...
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -230,11 +230,11 @@ Lass uns damit anfangen, Git tatsächlich zu verwenden. Der erste Schritt besteh
 <!--To install Git, you need to have the following libraries that Git depends on: curl, zlib, openssl, expat, and libiconv. For example, if you’re on a system that has yum (such as Fedora) or apt-get (such as a Debian based system), you can use one of these commands to install all of the dependencies:-->
 
-Um Git zu installieren, benötigst Du die folgenden Bibliotheken, die von Git verwendet werden: curl, zlib, openssl, expat und libiconv. Wenn Dir auf Deinem System yum (z.B. auf Fedora) oder apt-get (z.B. auf Debian-basierten Systemen) zur Verfügung steht, kannst Du einen der folgenden Befehle verwenden, um diese Abhängigkeiten zu installieren:
+Um Git zu installieren, benötigst Du die folgenden Bibliotheken, die von Git verwendet werden: curl, zlib, openssl, expat und libiconv. Wenn Dir auf Deinem System yum (z. B. auf Fedora) oder apt-get (z. B. auf Debian-basierten Systemen) zur Verfügung steht, kannst Du einen der folgenden Befehle verwenden, um diese Abhängigkeiten zu installieren:
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -244,7 +244,7 @@ Um Git zu installieren, benötigst Du die folgenden Bibliotheken, die von Git ve
 <!--When you have all the necessary dependencies, you can go ahead and grab the latest snapshot from the Git web site:-->
 
-Nachdem Du die genannten Bibliotheken installiert hast, besorge Dir die aktuelle Version des Git Quellcodes von der Git Webseite:
+Nachdem Du die genannten Bibliotheken installiert hast, besorg Dir die aktuelle Version des Git-Quellcodes von der Git-Webseite:
```
Consider changing `Webseite` to `Website`.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -283,7 +283,7 @@ Auf einem Debian-basierten System wie Ubuntu steht Dir apt-get zur Verfügung:
 <!--There are two easy ways to install Git on a Mac. The easiest is to use the graphical Git installer, which you can download from the SourceForge page (see Figure 1-7):-->
 
-Auf einem Mac kann man Git auf zwei Arten installieren. Der einfachste ist, das grafische Git Installationsprogramm zu verwenden, den man von der SourceForge Webseite herunterladen kann (siehe Bild 1-7)
+Auf einem Mac kann man Git auf zwei Arten installieren. Die einfachste ist, das grafische Git-Installationsprogramm zu verwenden, das man von der SourceForge-Webseite herunterladen kann (siehe Bild 1-7)
```
Consider appending a trailing colon `:` like in the source.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -317,17 +317,17 @@ Alternativ kann man Git auch über Homebrew (`http://brew.sh/`) installieren. Ha
 <!--Installing Git on Windows is very easy. The msysGit project has one of the easier installation procedures. Simply download the installer exe file from the GitHub page, and run it:-->
 
-Das msysGit Projekt macht die Installation von Git unter Windows sehr einfach. Lade einfach das Installationsprogramm für Windows von der GitHub Webseite herunter und führe es aus:
+Das msysGit Projekt macht die Installation von Git unter Windows sehr einfach. Lad einfach das Installationsprogramm für Windows von der GitHub-Webseite herunter und führe es aus:
```
This imperative has incorrect grammar in German. Verben mit Verbstammendung: `-t`, `-d`, `-ig`, `-er`, `-el` bekommen im Imperativ die Endung `-e`.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -346,54 +346,54 @@ Git umfasst das Werkzeug `git config`, das Dir erlaubt, Konfigurationswerte zu v
 <!--Now that your identity is set up, you can configure the default text editor that will be used when Git needs you to type in a message. By default, Git uses your system’s default editor, which is generally Vi or Vim. If you want to use a different text editor, such as Emacs, you can do the following:-->
 
-Nachdem Du Deine Identität jetzt konfiguriert hast, kannst Du einstellen, welchen Texteditor Git in Situationen verwenden soll, in denen Du eine Nachricht eingeben musst. Normalerweise verwendet Git den Standard-Texteditor Deines Systems – das ist üblicherweise Vi oder Vim. Wenn Du einen anderen Texteditor, z.B. Emacs, verwenden willst, kannst Du das wie folgt festlegen:
+Nachdem Du Deine Identität jetzt konfiguriert hast, kannst Du einstellen, welchen Texteditor Git in Situationen verwenden soll, in denen Du eine Nachricht eingeben musst. Normalerweise verwendet Git den Standard-Texteditor Deines Systems – das ist üblicherweise Vi oder Vim. Wenn Du einen anderen Texteditor, z. B. Emacs, verwenden willst, kannst Du das wie folgt festlegen:
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -346,54 +346,54 @@ Git umfasst das Werkzeug `git config`, das Dir erlaubt, Konfigurationswerte zu v
 <!--If you want to check your settings, you can use the `git config -\-list` command to list all the settings Git can find at that point:-->
 
-Wenn Du Deine Einstellungen überprüfen willst, kannst Du mit dem Befehl `git config --list` alle Einstellungen anzuzeigen, die Git an dieser Stelle (z.B. innerhalb eines bestimmten Projektes) bekannt sind:
+Wenn Du Deine Einstellungen überprüfen willst, kannst Du mit dem Befehl `git config --list` alle Einstellungen anzuzeigen, die Git an dieser Stelle (z. B. innerhalb eines bestimmten Projektes) bekannt sind:
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -406,7 +406,7 @@ Wenn Du Deine Einstellungen überprüfen willst, kannst Du mit dem Befehl `git c
 <!--You may see keys more than once, because Git reads the same key from different files (`/etc/gitconfig` and `~/.gitconfig`, for example). In this case, Git uses the last value for each unique key it sees.-->
 
-Manche Variablen werden möglicherweise mehrfach aufgelistet, weil Git dieselbe Variable in verschiedenen Dateien (z.B. `/etc/gitconfig` und `~/.gitconfig`) findet. In diesem Fall verwendet Git dann den jeweils zuletzt aufgelisteten Wert.
+Manche Variablen werden möglicherweise mehrfach aufgelistet, weil Git dieselbe Variable in verschiedenen Dateien (z. B. `/etc/gitconfig` und `~/.gitconfig`) findet. In diesem Fall verwendet Git dann den jeweils zuletzt aufgelisteten Wert.
```
See above.

jublo commented on de/01-introduction/01-chapter1.markdown:
```diff
@@ -416,11 +416,11 @@ Außerdem kannst Du mit dem Befehl `git config {key}` prüfen, welchen Wert Git
 <!--If you ever need help while using Git, there are three ways to get the manual page (manpage) help for any of the Git commands:-->
 
-Falls Du jemals Hilfe in der Anwendung von Git benötigst, gibt es drei Möglichkeiten, die entsprechende Seite aus der Dokumentation (manpage) für jeden Git Befehl anzuzeigen:
+Falls Du jemals Hilfe bei der Anwendung von Git benötigst, gibt es drei Möglichkeiten, die entsprechende Seite aus der Dokumentation (manpage) für jeden Git Befehl anzuzeigen:
```
Think about whether we might use `Nutzung` instead of `Anwendung` here?

jublo replied:
How about inserting a dash between `Git Befehl` (globally)?

jublo commented:
@vlt My compliment for a well-engineered review of chapter 1. Thank you for your hard work! Please find my comments inline.

vlt committed:
replaced SPACE with NARROW NO-BREAK SPACE (U+202F)

for German multi-part abbreviations (d. h., z. B.)
https://github.com/progit-de/progit/pull/13#discussion_r53634196

vlt committed:
Beteiligte is an adjective used as noun, so it needs to be capitalised

https://github.com/progit-de/progit/pull/13#discussion_r53634490

vlt committed:
concatenating "statt dessen" et. al.

https://github.com/progit-de/progit/pull/13#discussion_r53635418

vlt committed:
replaced "mächtigen" with "leistungsstarken"

https://github.com/progit-de/progit/pull/13#discussion_r53635916

vlt committed:
changed "Webseite" to "Website"

https://github.com/progit-de/progit/pull/13#discussion_r53636149

vlt committed:
appended a trailing colon like in the source

https://github.com/progit-de/progit/pull/13#discussion_r53636231

vlt committed:
correct imperative "lade"

https://github.com/progit-de/progit/pull/13#discussion_r53636484

Verben mit Verbstammendung: -t, -d, -ig, -er, -el
bekommen im Imperativ die Endung -e.

vlt committed:
"Nutzung" instead of "Anwendung"

https://github.com/progit-de/progit/pull/13#discussion_r53637893

vlt committed:
inserted a hyphen between "Git Befehl"

 https://github.com/progit-de/progit/pull/13#discussion_r53637954

vlt commented:
@jublo 
Hello!

Thank you for reviewing and adding to my changes in chapter 1.

I tried to correct my previous mistakes with the recent commits.

vlt commented:
As a side note:
While continuing reading on the weekend I discovered that there’s progit2.

This is mainly an exercise in git for me but do you think it would make more sense to continue with progit2?

Daniel
